commands/Moderation/unmute.js

```diff
@@ -0,0 +1,28 @@
+const { Command } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			permissionLevel: 6,
+			requiredPermissions: ['MANAGE_ROLES'],
+			runIn: ['text'],
+
+			description: 'Unmutes a mentioned user.',
+			usage: '<member:user> [reason:string] [...]',
+			usageDelim: ' '
+		});
+	}
+
+	async run(msg, [user, ...reason]) {
+		const member = msg.guild.members.get(user.id) || await msg.guild.members.fetch(user).catch(() => null);
+		if (member) {
+			if (member.roles.highest.position >= msg.member.roles.highest.position) throw 'You cannot unmute this user.';
+		}
+		if (!member.roles.has(msg.guild.configs.roles.muted)) throw 'This user is not muted.';
```

Comment:
if the member is null or undefined this will throw an error.

Reply:
same as above